dd-sdk-android-rx/src/main/kotlin/com/datadog/android/rx/DatadogRxExt.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rx
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Flowable
+import io.reactivex.rxjava3.core.Maybe
+import io.reactivex.rxjava3.core.Observable
+import io.reactivex.rxjava3.core.Single
+
+/**
+ * Returns an [Observable<T>] which intercepts any Exception thrown in the upper stream to
+ * send it as a RUM error event.
+ * The interceptor will not consume the error and you will be able to threat the error
+ * down the stream by using any of the available RxJava existing operators.
```

Comment:
Since the class is literally called `Consumer`, saying it doesn't consume the error is slightly weird. I think you actually can have a simpler documentation (and apply this to all the methods in this file ;) )

```suggestion
 * Returns an [Observable<T>] that will send a RUM Error event if this [Observable<T>] emits an error. Note that the error will also be emitted by the returned [Observable<T>].
```